husky_components/input/main.js

```diff
@@ -200,6 +200,13 @@ define([], function() {
          * Binds Dom-related events
          */
         bindDomEvents: function() {
+            this.sandbox.dom.on(this.$el, 'change', function(event) {
+                event.stopPropagation();
+                event.preventDefault();
+
+                this.$el.trigger('change');
```

Comment:
you have overseen the filter in the next line (: i catch the event from the input not from the $el!